daemon/logger/factory.go

```diff
@@ -46,7 +53,7 @@ func (lf *logdriverFactory) get(name string) (Creator, error) {
 
 	c, ok := lf.registry[name]
 	if !ok {
-		return c, fmt.Errorf("logger: no log driver named '%s' is registered", name)
+		return c, fmt.Errorf("unsupported log driver '%s'", name)
```

Comment:
@thaJeztah in theory it's indeed `unsupported` because the log drivers is _installed_ at build time, not runtime.
To make jenkins happy I'll restore the error message in this PR.